src/python/dxpy/scripts/dx.py

```diff
@@ -2514,7 +2514,15 @@ def install(args):
 def uninstall(args):
     app_desc = get_app_from_path(args.app)
     if app_desc is None:
-        parser.exit(1, 'Could not find the app\n')
+        try:
+            dxpy.api.app_uninstall(args.app)
```

Comment:
From @ptn24 : Rather than simply invoking /app-x/uninstall here, perhaps we can verify whether the specified app is present in the appsInstalled field of the self describe response, and then only uninstall if present. Otherwise, we can raise an error and exit, as before. Does this make sense at all?